### DISCUSSION TITLE
Seperate arena for only Heuristic snakes

### DISCUSSION THREAD

**Discussion by FreeThrill** [BattlesnakeOfficial/feedback#146] [Feature Requests]
I noticed that almost all of the top snakes use machine learning. While this is great, there are some school kids like me who don't have access to Machine Learning resources, and are not ready to compete against professional software engineers who formed the top teams. Therefore, I would like to form a separate arena for Heuristic snakes only. Then it would be a lot more fun for Heuristic based snakes to compete in a level playing field.

**Comment by InterestingBrainPoops** [comment]
Very few of the top snakes use machine learning, do you mean tree search / minimax / MCTS?

**Comment by bvanvugt** [comment]
I agree with @InterestingBrainPoops that most top-tier snakes aren't actually machine learning algorithms, but tend towards complex and compute expensive heuristics. That said...

I think this is a valid point, and thanks for making it @FreeThrill :) Maybe a low-stakes rookie/student league or tournament is a good solution here. Open to other ideas as well!
